crypto/bio/bss_dgram.c

```diff
@@ -294,26 +308,262 @@ static void dgram_reset_rcv_timeout(BIO *b)
 # endif
 }
 
+# if defined(HAVE_IP_PKTINFO)
```

Comment:
Rather than creating a custom define here, could we not just use `__linux__`? Adding new defines to Configure just for this seems quite heavyweight.